static/docs/commands-reference/config.md

```diff
@@ -41,156 +45,163 @@ share through Git.
 
 * `-v`, `--verbose` - displays detailed tracing information.
 
-## core
+## Available setting names and sections
+
+These are the `name` parameters that can be used with `dvc config`, or the
+sections in the project config file (`.dvc/config`).
+
+### remote
+
+These are sections in the config file that describe particular remotes. See `dvc
+remote` for more info.
+
+### core
+
+This is the main section with the general settings:
+
+* `core.loglevel` - log level that the `dvc` command should use. Possible values are
+  `info`, `debug`, `warning`, or `error`.
+
+* `core.remote` - name of the remote that should be used by default.
+
+### cache
+
+DVC cache is a hidden storage (by default located in the `.dvc/cache` directory)
+for files that are under DVC control, and their different versions. (See `dvc
+cache` for more details.)
+
+* `cache.dir` - set/unset cache directory location. A correct value must be either an
+  absolute path or a path **relative to the config file location**. The default
+  value is `cache`, which resolved relative to the default project config
+  location results in `.dvc/cache`.
+  > See also helper command `dvc cache dir` that properly transform paths
+  relative to the present working directory into relative to the project config
+  file.
+
+* `cache.protected` - makes files in the workspace read-only. Possible values
```

Comment:
should mention that if `hardlink` or `symlink` cache type is used it's highly recommend to enable this mode.

Reply:
Added